### ISSUE TITLE
Permissions Question

### ISSUE THREAD

**Issue by robertbur** [Requarks/wiki#2506]
**Question**
Hello everyone,

I have a question regarding the permissions. Looks like I set correctly all the permissions for a group, however, when a user is going to create a new page, he can't modify the Style at the website. 

Have a look at these screenshots:

<img width="350" alt="Style2" src="https://user-images.githubusercontent.com/37327232/94362703-88ddfa80-00bd-11eb-9e20-53cd8e9f0741.PNG">
<img width="752" alt="Style" src="https://user-images.githubusercontent.com/37327232/94362708-8c718180-00bd-11eb-88e5-cd74918dfb9b.PNG">

**Host Info (please complete the following information):**
OS: [Debian 10, Docker]
Wiki.js version: [2.5.144]
Database engine: [Maria DB 10]

Thanks in advance ;)

**Comment by Smankusors** [comment]
How about page rules? [This documentation](https://docs.requarks.io/groups) could helpful though.

**Comment by NGPixel** [comment]
As @Smankusors mentionned, you also need to define Page Rules. Read the docs on how it works.

**Comment by robertbur** [comment]
Hello Guys,

Sorry, but I did a test and it doesn't work. I gave full access to the folder to the user at the pages rules to all the options, and when he creates a new page, the Style option doesn't appear. 

My page Rules:

<img width="1195" alt="test" src="https://user-images.githubusercontent.com/37327232/94411946-7cb77300-0179-11eb-85ae-b5165e3b2fe3.PNG">

I also checked the link that @Smankusors sent to me.

Thanks in advance

Kind Regards
